xr_sample_project/Assets/SampleViewer/Samples/VRSample/Scripts/VRMenuManager.cs

```diff
@@ -29,7 +39,7 @@ private void Update()
     {
         if (esriMenu)
         {
-            if (toggleMenuButton.action.WasPressedThisFrame() && !currentlyTeleporting)
+            if (toggleMenuButton.triggered)
```

Comment:
Is this shared between the two samples? Does removing the `currentlyTeleporting` check still work in the VR sample?

Reply:
So there are two reasons for this file be edited. For one, when I imported my sample, there were compile errors from the VR Sample. Idk what happened/broke but this was the fix in order for me to get the basics working. Secondly, I will be going through and redoing the VR Sample at some point in the future to give the UI updates and the same "treatment" as Unreal VR Sample. Even in it's current state the VR sample is heavily broken on Main. I believe it works still but some functionality that should be there does not work and idk why. It is likely due to how the sample was imported/exported when we split the sample project.